### ISSUE TITLE
Error: undeclared identifier: 'MediaType'

### ISSUE THREAD

**Issue by adrianperreault** [status-im/nim-json-rpc#145]
Hi, I'm trying to run:
``` nim
import json_rpc/rpcserver

var router = newRpcRouter()

router.rpc:
  result = %"Hello"
```

But I run into: `nim-chronos/chronos/apps/http/httpcommon.nim(35, 28) Error: undeclared identifier: 'MediaType'`

Thx

**Comment by mratsim** [comment]
It seems like we have another generic sandwich issue https://github.com/nim-lang/Nim/issues/11225

In the mean time you can solve that with `import httputils` for https://github.com/status-im/nim-http-utils/blob/e88e231/httputils.nim#L340-L342

**Comment by jangko** [comment]
The snippet is super outdated,  using this should ok. fixed by #178, closing.

```Nim
import json_rpc/rpcserver

var router = RpcServer.new()

router.rpc("something") do():
  return %"Hello"
```
